src/game_api/flags.hpp

```diff
@@ -754,4 +754,5 @@ const char* pause_types[]{
     "8: ?",
     "16: ?",
     "32: Ankh (smooth camera, janky audio)",
+    "Freeze on PRE_UPDATE",
```

Comment:
missing the actual flag number?

Reply:
Not a real flag, just a way to pause.

Reply:
So only the UI thing? maybe add a comment about this, since we link the `flags.hpp` in the API doc as well